int32           selectedUAV
float64[]       building_x
float64[]       building_y
float64         flight_altitude
float64         building_distance
float64         trajectory_resolution
string          wp_file_path
string		    coord_sys_text
bool		    plot_flag
float64         h_speed
float64         v_speed	
float64         h_acc	
float64         v_acc